internal/compiler/tests/syntax/lookup/issue_1461.slint

```diff
@@ -0,0 +1,23 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import { Button, VerticalBox } from "std-widgets.slint";
+ 
+ export AppWindow := Window {
+     property<int> counter: 42;
+     callback request-increase-value();
+     VerticalBox { 
+         Button {
+             text: "Increase value";
+             clicked => { 
+                 request-increase-value();
+             }
+             states [
+                 highlight when counter > 45 : {
+                     background: red;
+//                   ^error{Internal error: The expression for the default state currently cannot be represented: https://github.com/slint-ui/slint/issues/1461}
```

Comment:
I’m curious: why is counter > 45 the default state? Why can’t background or counter be referenced? Is there a component boundary that I’m missing?

Reply:
`counter > 45`  is not the default state.  The problem is that the lowered binding can't represent the binding for the default state (not this one) because it needs access to properties which cannot be named from that context.

Reply:
Ahh. I'm curious, why can't it be represented in this position?

Reply:
I updated the description of #1490 with a more detailed explaination

Reply:
Thanks - now I understand!